internal/service/resourcepolicy/resource_test.go

```diff
@@ -33,16 +41,31 @@ var (
 	) when {
 	context.cluster.cloudProviders.containsAny([cloud::cloudProvider::"aws"])
 	};`
+	validPolicyProjectForbidIPAccessAnywhere = `
+	forbid (
+		principal,
+		action == cloud::Action::"project.edit",
+		resource
+	) 
+		when {
+		context.project.ipAccessList.contains(ip("0.0.0.0/0"))
+	};`
 )
 
 func TestAccResourcePolicy_basic(t *testing.T) {
+	tc := basicTestCase(t)
+	resource.Test(t, *tc)
+}
+
+func basicTestCase(t *testing.T) *resource.TestCase {
+	t.Helper()
 	var (
 		orgID       = os.Getenv("MONGODB_ATLAS_ORG_ID")
 		policyName  = "test-policy"
 		updatedName = "updated-policy"
 	)
 
-	resource.ParallelTest(t, resource.TestCase{
+	return &resource.TestCase{
```

Comment:
non-blocking: what not parallel anymore? can you share the context?

Reply:
Good point. Updated in 8a3cf3a6eb25a65e4733c37176cc04a979c62720 (Need sequential execution for assertions to be deterministic (plural data source))

Reply:
@EspenAlbert have you tried to use depends_on the resource in singular & plurar ds?

Reply:
How would that differ? (right now the implicit dependencies are resource <-- data-source <-- data-source-plural)
The problem is that the resource-policies are created within the same Organization.
Therefore, the plural data source might read more than 1 policy if tests are running in parallel.

Reply:
singular ds seems to depend on resource if policy id is computer. However plural ds doesn't use any resource computed attribute so can be run at any time, even before the resource.    but i understand the dependency you say with other tests in that org

Reply:
In practice, I think it doesn't matter that the org_id is not computed. The test would fail if the plural data source tried reading immediately.